apps/dashboard/src/@3rdweb-sdk/react/hooks/useTransferNativeToken.tsx

```diff
@@ -8,13 +15,23 @@ type TransferInput = {
   amount: string;
 };
 
-export function useTransferNativeToken() {
-  const sdk = useSDK();
+export function useTransferNativeToken({
```

Comment:
I'm *sure* we have a hook for this in `thirdweb/react` - especially since the connect UI has a way to do this, might have to expose it if it's not exported (add docs if you do so) but it definitely exists